runtime/filetype.vim

```diff
@@ -3140,4 +3140,8 @@ endfunc
 let &cpo = s:cpo_save
 unlet s:cpo_save
 
+" Stylus
+autocmd BufNewFile,BufReadPost *.styl set filetype=stylus
+autocmd BufNewFile,BufReadPost *.stylus set filetype=stylus
```

Comment:
Please move up and follow the style used elsewhere. And when you add a new entry to filetype.vim, please also add a test for it in testdir/test_filetype.vim

Reply:
Fixed here: https://github.com/vim/vim/pull/14656/commits/73250e9474c45bd1a46816e641d10b9941f1a589